corehq/apps/app_manager/remote_app.py

```diff
@@ -81,9 +81,6 @@ def make_remote_profile(app):
                     download_index_url=download_index_url
                 )
 
-        if app.build_langs:
-            profile_xml.set_property("cur_locale", app.build_langs[0])
-
```

Comment:
It looks to me like the mistake was to delete the second line as well and this should apply to all apps (using `app.langs[0]`), unless there is a situation in which there are no build langs. Is that the case for Bihar? If so, what is the expected behavior? The idea was that remote apps would not support build profiles, but if we need to handle an app with no languages I can work that in.